SU-SWS/stanford_capx
# PHP7.2 Compatibility.

# READY

# Summary
- PHP 7.2 Compatibility

# Needed By (Date)
- Week of January 21st

# Urgency
- Medium: Has impact on ACE and ACSF migrations.

# Steps to Test

1. Check out this branch on a server with PHP 7.2 enabled
2. Run a JSE install
3. Scan logs for errors
4. Run an importer
5. Ensure that all data was imported into the mapped entities 
---
1. Check out default 7.x-3.x branch on an environment with php 7.1 or less running
2. Install JSE profile
3. Add valid CAPx credentials and save form.
4. Configure importer to pull in BUWS org
5. Run importer and import people
6. Check out this branch
7. Run `drush updb`
8. Delete all imported profiles
9. Run importer again
10. Validate profiles have imported
11. Reload webserver with php 7.2 
12. Run importer 

# Affected Projects or Products
- Everyone

# Associated Issues and/or People
- ?
